### DISCUSSION TITLE
Infinite scroll over pagination

### DISCUSSION THREAD

**Discussion by Khushli** [twilio-labs/paste#2010] [Office hours] [answered]
Hi,

We have a long detailed log tables in the accounts insights page (Console). We are trying to use pagination component but the engineering team is running into a technical roadblock where the user can only go forward to the next page but not go back to a pervious page. The experience is becoming quite unfriendly because of this. 

I am looking for some guidance on the possibility of using infinity scroll instead. Infinity scroll will provide a much better user experience in this case where pagination is technically not working. 

Thanks

**Comment by nkrantz** [comment]
Hi @Khushli,

Could you please provide more context on what the technical roadblock with Pagination is? Pagination is composable, so you're able to use only forward arrows (without back arrows) if that's what you need. It's hard to provide guidance without more context, so feel free to also [sign up for an Office Hours slot](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) to discuss this synchronously.

Thanks!

**Comment by Khushli** [comment]
Thanks for the response. We know that pagination can only be used with forward arrows. But the experience we want is to help users easily move back and forth, since pagination is not allowing that we are thinking of using infinity scroll. 

I don't know how to tag the engineering team here but they should be able explain the reason for not being able to implement backward pagination. I've invited an engineer to the OH to help explain the issue. 

@katieporter want to discuss this as well in our OH

**Reply by nkrantz** [reply]
Sounds like we'll address these issues in Office Hours. Please make sure to [sign up for a slot](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) so we can make sure that we have enough designers and engineers present to tackle the technical issues as well as any potential solutions. I'll switch the category of this Discussion to "Office Hours." Thanks!

**Comment by kate-porter** [comment, marked as answer]
Hey @Khushli! Just wanted to share the [link from last week's office hours](https://www.loom.com/share/a511fd0e9bf54f9687ab1b093dd18026) in case you or Jack would like to reference anything we discussed. The password for the video is in the #help-design-system Slack channel.

If you have any other questions come up around this, please let us know!
